blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeImpl.java

```diff
@@ -169,4 +194,35 @@ FlowNodeWrapper getFlowNodeWrapper(){
         return node;
     }
 
+    @Extension
+    public static class NodeFactoryImpl extends BlueTestResultFactory {
```

Comment:
This class looks specific to JUnit - perhaps it should move into `BlueJUnitTestResult`?